.github/workflows/build-extension-jar.yml

```diff
@@ -0,0 +1,117 @@
+        type: string
+
+
+jobs:
+  delete-dependency-packages:
+    runs-on: ubuntu-22.04
+    continue-on-error: true
+    steps:
+      - uses: actions/delete-package-versions@v5
+        with:
+          package-name: org.liquibase.ext.${{ inputs.extension }}
+          package-type: 'maven'
+          token: ${{ secrets.BOT_TOKEN }}
+          min-versions-to-keep: 5 # Keep the last 5 versions so we don't break simultaneous builds
+          delete-only-pre-release-versions: true
+
+  build-and-deploy-extensions:
+    needs: [delete-dependency-packages]
+    runs-on: ubuntu-22.04
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      GIT_USERNAME: "liquibot"
+      GIT_PASSWORD: ${{ secrets.LIQUIBOT_PAT_GPM_ACCESS }}
+    steps:
+      - name: Checkout Dependencies
+        run: |
+            git config --global credential.helper store
+            echo "https://$GIT_USERNAME:$GIT_PASSWORD@github.com" > ~/.git-credentials
+            git clone https://github.com/liquibase/${{ inputs.extension }}.git ${{ inputs.extension }}
+
+      - name: Set up JDK 21
+        uses: actions/setup-java@v4
+        with:
+          java-version: 21
+          distribution: 'temurin'
+          cache: 'maven'
+          gpg-private-key: ${{ secrets.GPG_SECRET }}
+          gpg-passphrase: GPG_PASSPHRASE
+        env:
+           GPG_PASSWORD: ${{ secrets.GPG_PASSPHRASE }}
+
+        #look for dependencies in maven
+      - name: maven-settings-xml-action
+        uses: whelk-io/maven-settings-xml-action@v22
+        with:
+          repositories: |
+            [
+              {
+                "id": "liquibase",
+                "url": "https://maven.pkg.github.com/liquibase/liquibase",
+                "releases": {
+                  "enabled": "true"
+                },
+                "snapshots": {
+                  "enabled": "true",
+                  "updatePolicy": "always"
+                }
+              },
+              {
+                "id": "liquibase-pro",
+                "url": "https://maven.pkg.github.com/liquibase/liquibase-pro",
+                "releases": {
+                  "enabled": "true"
+                },
+                "snapshots": {
+                  "enabled": "true",
+                  "updatePolicy": "always"
+                }
+              }
+            ]
+          servers: |
+            [
+              {
+                "id": "liquibase",
+                "username": "liquibot",
+                "password": "${{ secrets.LIQUIBOT_PAT_GPM_ACCESS }}"
+              },
+              {
+                "id": "liquibase-pro",
+                "username": "liquibot",
+                "password": "${{ secrets.LIQUIBOT_PAT_GPM_ACCESS }}"
+              }
+            ]
+
+      - name: Build and deploy Extension to GPM
+        env:
+           GPG_PASSWORD: ${{ secrets.GPG_PASSPHRASE }}
+        run: |
+          cd ${{ inputs.extension }}
+          EXTENSION_VERSION="master-SNAPSHOT"
```

Comment:
is this version correct?

Reply:
Yes... I defined master-SNAPSHOT as this is the one that we publish in core. But it could be moved to an action input using master-SNAPSHOT as the default value, what do you think?

Reply:
I guess this is for dev porpouses right? see my comment here https://datical.atlassian.net/browse/DAT-18367?focusedCommentId=146471

Reply:
Yes, all for dev and not related to release process. We need to make dev work together again after the new requirements from release, currently both don't work together.

Reply:
I will take a look and see if we can call this from liquibase

Reply:
I wasn't thinking about reusing it on liquibase-core, but now that you mentioned it... I think we can change it to use to build the extensions if we make the EXTENSION_VERSION a parameter :palms_up_together: